README.md

```diff
@@ -4,9 +4,21 @@
 
 Runly is an opinionated framework for writing multi-threaded jobs. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.
```

Comment:
```suggestion
Runly is a toolset for simplifying complex applications by writing compartmentalized, testable code. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.
```

Reply:
I think we should leave "opinionated framework" in there. It conveys that we've though of the issues you are going to have before you have and have designed the API in a way that will give you the least friction ([stolen from here](https://stackoverflow.com/questions/802050/what-is-opinionated-software#:~:text=An%20opinionated%20framework%20is%20one,made%20by%20the%20framework%20designer.%20%E2%80%93)). Maybe we could spell that out more explicitly if you want?

Reply:
I know this is a thing, but really everyone has an opinion that shows in their code. I feel like we don't need to hit people over the head with this, it should be obvious, and I wouldn't want to turn people off right away who look at opinionated software as bad software.